pkg/operation/care/health.go

```diff
@@ -126,16 +120,7 @@ func (h *Health) Check(
 	return PardonConditions(h.clock, updatedConditions, lastOp, lastErrors)
 }
 
-// ExtensionCondition contains information about the extension type, name, namespace and the respective condition object.
-type ExtensionCondition struct {
-	Condition          gardencorev1beta1.Condition
-	ExtensionType      string
-	ExtensionName      string
-	ExtensionNamespace string
-	LastHeartbeatTime  *metav1.MicroTime
-}
-
-func (h *Health) getAllExtensionConditions(ctx context.Context) ([]ExtensionCondition, []ExtensionCondition, []ExtensionCondition, error) {
+func (h *Health) getAllExtensionConditions(ctx context.Context) ([]healthchecker.ExtensionCondition, []healthchecker.ExtensionCondition, []healthchecker.ExtensionCondition, error) {
```

Comment:
```suggestion
func (h *health) getAllExtensionConditions(ctx context.Context) ([]healthchecker.ExtensionCondition, []healthchecker.ExtensionCondition, []healthchecker.ExtensionCondition, error) {
```
the data type could be `health` instead of `Health` here too.